### PR TITLE
Lesson 5 Assignment

### PR DESCRIPTION
**Pull request by sonalaggarwal555**

### CODE REVIEW

**Comment by cherimarie** [comment]
@sonalaggarwal555 I can't review a PR with 1,692 files changed. Can we please clean this up so that the only file changes are those directly related to the assignment?

**Comment by sonalaggarwal555** [comment]
@cherimarie I have made changes in the assignment.I have deleted all fontawesome files and added them to node modules. I mistakenly merged the PR . But let me know if you want me to revert this PR and reopen the same.

**Comment by sonalaggarwal555** [comment]
Hi Cheri,
I have updated the PR.Here is the link:
https://github.com/sonalaggarwal555/html300/pull/4/files .Please review the
changes.

**Comment by sonalaggarwal555** [comment]
Hello Cheri,

Due to some mistake,i had to delete 'assignment 06' branch and create it
again.I have updated my assignment in that branch and have generated a new
pull request.Please find the below PR link and ignore the one i have
submitted in the assignment.
https://github.com/sonalaggarwal555/html300/pull/6

Thanks,
Sonal

File: lesson05/assignment/css/style.scss [hunk 1]
```diff
@@ -3,4 +3,94 @@
...
+main{
+  display:grid;
+  grid-template-rows:auto auto auto auto;
+}
+header{
+  background-color: rgb(241, 53, 43);
+  width:100%;
+  h1{
+  color: white;
+  font-family: HelveticaNeue, "Helvetica Neue";
```
Comment (cherimarie) [inline lesson05/assignment/css/style.scss:16]: Font family list should be in descending order of specificity, so this should look something like: 
` font-family: "Helvetica Neue",  Helvetica, Arial, sans-serif;`
Give lots of fallback options.

File: lesson05/assignment/index.html [hunk 2]
```diff
@@ -1,15 +1,117 @@
...
-    </body>
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="node_modules/@fortawesome/fontawesome-free/css/all.min.css"/>
+    <link rel="stylesheet" href="node_modules/@fortawesome/fontawesome-free/css/brands.min.css"/>
+    <link rel="stylesheet" href="node_modules/@fortawesome/fontawesome-free/css/solid.css"/>
+    <link rel="stylesheet" href="node_modules/@fortawesome/fontawesome-free/css/fontawesome.min.css"/>
+    <link rel="stylesheet" href="node_modules/@fortawesome/fontawesome-free/css/solid.min.css">
```
Comment (cherimarie) [inline lesson05/assignment/index.html:11]: 👍

**Pull request merged**